app/views/houseConsignment/index/departureMeansOfTransport/IdentificationNumberView.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.MovementReferenceNumber
+@import viewModels.components.InputTextViewModel._
+@import views.html.components.{Button, ErrorSummary, InputText}
+@import controllers.houseConsignment.index.departureMeansOfTransport.routes
+@import viewModels.departureTransportMeans.IdentificationNumberViewModel
+
+@this(
+    mainTemplate: MainTemplate,
+    formHelper: FormWithCSRF,
+    button: Button,
+    inputText: InputText,
+    errorSummary: ErrorSummary
+)
+
+@(form: Form[_], mrn: MovementReferenceNumber, arrivalId: ArrivalId, houseConsignmentIndex: Index, transportMeansIndex: Index, mode: Mode, viewModel: IdentificationNumberViewModel)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(
+    title = viewModel.title,
+    showBackLink = true,
+) {
+
+    @errorSummary(form.errors)
+
+    @formHelper(action = routes.IdentificationNumberController.onSubmit(arrivalId,houseConsignmentIndex, transportMeansIndex, mode), Symbol("autoComplete") -> "off") {
+
+
+        @inputText(
+            field = form("value"),
+            inputTextType = OrdinaryTextInput(
+                heading =  viewModel.heading,
+                caption = Some(messages("site.caption", mrn.toString))
+            ),
+            hint = Some(messages("departureMeansOfTransport.identificationNumber.hint"))
```

Comment:
better to have the prefix as below - as its under house-consignment level houseConsignment.index.departureMeansOfTransport.identificationNumber

looks like content messages are bit different from content pages - better to have a separate messages  instead of using messages from consignment level